[2022-02-11 11:20:06,871] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_workflow.test_function scheduled__2022-02-08T00:00:00+00:00 [queued]>
[2022-02-11 11:20:06,969] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_workflow.test_function scheduled__2022-02-08T00:00:00+00:00 [queued]>
[2022-02-11 11:20:06,973] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-11 11:20:06,977] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-11 11:20:06,983] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-11 11:20:07,041] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): test_function> on 2022-02-08 00:00:00+00:00
[2022-02-11 11:20:07,056] {standard_task_runner.py:52} INFO - Started process 63 to run task
[2022-02-11 11:20:07,079] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_workflow', 'test_function', 'scheduled__2022-02-08T00:00:00+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/test_workflow.py', '--cfg-path', '/tmp/tmp0jc4clv0', '--error-file', '/tmp/tmpot6cla9f']
[2022-02-11 11:20:07,083] {standard_task_runner.py:77} INFO - Job 43: Subtask test_function
[2022-02-11 11:20:07,357] {logging_mixin.py:109} INFO - Running <TaskInstance: test_workflow.test_function scheduled__2022-02-08T00:00:00+00:00 [running]> on host 692d64867305
[2022-02-11 11:20:07,569] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_workflow
AIRFLOW_CTX_TASK_ID=test_function
AIRFLOW_CTX_EXECUTION_DATE=2022-02-08T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-08T00:00:00+00:00
[2022-02-11 11:20:07,592] {python.py:175} INFO - Done. Returned value was: None
[2022-02-11 11:20:07,637] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=test_workflow, task_id=test_function, execution_date=20220208T000000, start_date=20220211T112006, end_date=20220211T112007
[2022-02-11 11:20:07,796] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-11 11:20:07,925] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
